file-formats/aifd/examples/z_aff_example_aifd.aifd.json

```diff
@@ -0,0 +1,22 @@
+{
+  "formatVersion": "1",
+  "header": {
+    "description": "Example AIFD for ABAP File Format",
+    "originalLanguage": "en",
+    "abapLanguageVersion": "cloudDevelopment"
+  },
+  "aifInterfaces":[
+    {
+      "objectName":"/AIFTCUST_WS_I00001",
+      "namespace":"/AIFT",
+      "interfaceName":"CUST_WS_I",
+      "interfaceVersion":"00001"
+    },
+    {
+      "objectName":"/AIFTCUST_WS_O00001",
+      "namespace":"/AIFT",
+      "interfaceName":"CUST_WS_O",
+      "interfaceVersion":"00001"
```

Comment:
Just one more question: Since only `objectName` is the only required field, `namespace`, `interfaceName` and `interfaceVersion` can be empty. Is this the behavior you intend?

Reply:
No, the fields namespace, interfaceName and interfaceVersion should be filled, but not changeable. The fields should be filled automatically by filling the objectName. The fields can be used for the objectName search, as the value of the ObjectName may be a generated and not readable value, so the fields are needed to identify the assigned AIF interfaces.

Reply:
I had a call with @D047539: the important field here is the `interfaceObjectName`/`objectName`. This field does not need to be derived from the other 3 fields. These fields, however, can help to better identify the chosen object. But they are not needed for the AIFD object. Hence, we decided to remove these fields and only keep the necessary field `interfaceObjectName`/`objectName`.